weblab/experiments/tests/test_views.py

```diff
@@ -1128,7 +1128,7 @@ def test_experiment_json(self, client, logged_in_user, experiment_version):
         version.author.full_name = 'test user'
         version.author.save()
         version.status = 'SUCCESS'
-        version.abs_path.mkdir()
+        # version.mkdir()
 
```

Comment:
did you mean to comment this out

Reply:
Ah, no, it should be deleted! I was just commenting it out while testing to make sure it wasn't needed.